src/Previewer/previewer.vala

```diff
@@ -59,10 +59,15 @@ namespace Workbench {
       return true;
     }
 
-    public void update_ui (string content, string target_id, string original_id = "") {
+    // This registers GObjects so that they can be found by GtkBuilder
+    void ensure_types() {
```

Comment:
This should have the `private` modifier.

Reply:
Private is not the default?
Then why do we put public on all the other functions?

Reply:
It is, but it should be explicit.